WalletWasabi.Fluent/Views/Wallets/WalletView.axaml

```diff
@@ -26,13 +26,17 @@
                 <TextBlock Text="Broadcast" />
```

Comment:
The `Broadcast` button is visible for those HW wallets, who prefer to sign the transaction without connecting the wallet to the machine. So they can create a file in the Send, they can send it here by pressing on the Broadcast.
Shouldn't this be hidden also if the `IsWalletBalanceZero` is true?

Reply:
I missed that one, will fix.

Reply:
Pushed changes.